Solution.py

```diff
@@ -0,0 +1,43 @@
+class Solution:
+    def EvenSum (list):
```

Comment:
- Good job using modulo operator
- Its more pythonic to use a list comprehension for this